README.md

```diff
@@ -166,9 +166,7 @@ codeInsights:
   ## map the values used in your code to the corresponding variable key in DevCycle
   variableAliases:
     "VARIABLES.ENABLE_V1": "enable-v1"
-  ## fully override the regex patterns used to match variables for a specific file extension
-  ## each pattern should contain exactly one capture group which matches on the key of the variable
-  ## make sure the captured value contains the entire key parameter (including quotes, if applicable)
+  ## additional regex patterns used to match variables for a specific file extension
```

Comment:
is it no longer the case that this setting overrides the default patterns rather than supplementing them?

Reply:
they've always been used in addition to the default match patterns